events/scripts/past-events.js

```diff
@@ -106,44 +106,50 @@ document.getElementById("past-events-details").innerHTML = htmlCode;
 
 // for the Upcoming event automation 
 var upcomeEventHtmlCode = ``;
+
+var checkDate = (eventDate) => {
+  var TodayDate = new Date().getTime();
+  var eveDate = new Date(eventDate).getTime();
+  if (TodayDate >= eveDate) {
+    return true;
+  }
+  return false;
+}
+
+var CommEventToggle = checkDate(mainEvent[1].dateComEvent) ? 1 : 0; // true for fresh && false for Stale community event
```

Comment:
![image](https://user-images.githubusercontent.com/78547746/187238998-1eebd957-d9fe-495e-8fe5-2c0cd29c0b2c.png)

function is not working properly on the input of date in string format. kindly see. also the function will return true for events which are stale/expired as per the current logic. however you are assigning "1" to such events which should not be the case. kindly check that too